### PR TITLE
Try fix codecov coverage

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

**Comment by JayH5** [comment]
As far as I can tell, what happened here is that when codecov runs and there is no `coverage.xml` file, it tries to generate one from the existing coverage information (somehow). This breaks because the `.coverage` file uses absolute paths, which are all paths for inside the Docker container, not the Travis build environment.

Getting coverage itself to generate the `coverage.xml` fixes things as codecov can basically just upload that file.

File: .travis.yml [hunk 1]
```diff
@@ -16,11 +16,10 @@ install:
   - pip install codecov
 
 before_script:
-  - if [[ -z "$NO_COVERAGE" ]]; then COV_OPT='--cov'; fi
```
Comment (jerith) [inline .travis.yml:19]: Is coverage unconditional now? Should it be?

Comment (JayH5) [reply .travis.yml:19]: It has been unconditional for a while. I didn't test extensively, but the speed gains without coverage on PyPy seemed very small and not worth the added complexity.

**Review by jerith — APPROVED** [review]
:+1:

**Pull request merged**